travis/test_server.py

```diff
@@ -296,11 +304,33 @@ def main(argv=None):
     counted_errors = 0
     for to_test in to_test_list:
         print("\nTesting %s:" % to_test)
-        subprocess.call(["createdb", "-T", dbtemplate, database])
+        db_odoo_created = False
+        try:
+            db_odoo_created = subprocess.call(
+                ["createdb", "-T", dbtemplate, database])
+        except subprocess.CalledProcessError:
+            db_odoo_created = True
         for command, check_loaded in commands:
-            command[-1] = to_test
-            # Run test command; unbuffer keeps output colors
-            command_call = ["unbuffer"] + command
+            if db_odoo_created and instance_alive:
```

Comment:
Can we drop in a `createdb` command somewhere to make that part of the code run?